history/characters/10000_orc.txt

```diff
@@ -1086,12 +1086,13 @@
 # Garona
 10620={
 	name=Garona
+	dynasty=2758
 	female=yes
 	dna="aelefapjdmh"
 	culture=shadowmoon religion=orcish_fel
 	martial=6 diplomacy=4 stewardship=8 intrigue=5 learning=8
 	add_trait=elusive_shadow
-	add_trait=content add_trait=cynical add_trait=trusting add_trait=humble add_trait=patient add_trait=chaste 
+	add_trait=content add_trait=cynical add_trait=trusting add_trait=humble add_trait=patient add_trait=chaste add_trait = legit_bastard
```

Comment:
Why she is a bastard?

Reply:
her parents werent married. her dad raped her mom

Reply:
Case closed!